homeassistant/components/switchbot_cloud/sensor.py

```diff
@@ -0,0 +1,93 @@
+"""Platform for sensor integration."""
+
+from switchbot_api import Device, SwitchBotAPI
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE, UnitOfTemperature
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import SwitchbotCloudData
+from .const import DOMAIN
+from .coordinator import SwitchBotCoordinator
+from .entity import SwitchBotCloudEntity
+
+SENSOR_TYPE_TEMPERATURE = "temperature"
+SENSOR_TYPE_HUMIDITY = "humidity"
+SENSOR_TYPE_BATTERY = "battery"
+
+METER_PLUS_SENSOR_DESCRIPTIONS = (
+    SensorEntityDescription(
+        key=SENSOR_TYPE_TEMPERATURE,
+        name="Temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+    ),
+    SensorEntityDescription(
+        key=SENSOR_TYPE_HUMIDITY,
+        name="Humidity",
+        device_class=SensorDeviceClass.HUMIDITY,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+    ),
+    SensorEntityDescription(
+        key=SENSOR_TYPE_BATTERY,
+        name="Battery",
```

Comment:
```suggestion
        device_class=SensorDeviceClass.TEMPERATURE,
        state_class=SensorStateClass.MEASUREMENT,
        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
    ),
    SensorEntityDescription(
        key=SENSOR_TYPE_HUMIDITY,
        device_class=SensorDeviceClass.HUMIDITY,
        state_class=SensorStateClass.MEASUREMENT,
        native_unit_of_measurement=PERCENTAGE,
    ),
    SensorEntityDescription(
        key=SENSOR_TYPE_BATTERY,
```

It will use the device class name